extensions.cabal

```diff
@@ -1,6 +1,6 @@
 cabal-version:       2.4
 name:                extensions
-version:             0.1.0.3
+version:             0.1.0.4
```

Comment:
This is a behaviour change so needs a bump in the third component. But no need to bump here. I'll do it before I release.

```suggestion
```